patterns/testimonials-three-columns-six-testimonials.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Title: Three column layout with 6 testimonials
+ * Slug: twentytwentyfive/testimonials-three-columns-six-testimonials
+ * Keywords: testimonial
+ * Categories: testimonials
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"},"blockGap":"var:preset|spacing|50"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull" style="padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)"><!-- wp:heading {"align":"wide","fontSize":"xx-large"} -->
+<h2 class="wp-block-heading alignwide has-xx-large-font-size"><?php echo esc_html_x( 'What people are saying', 'Testimonial section heading', 'twentytwentyfive' ); ?></h2>
+<!-- /wp:heading -->
+
+<!-- wp:columns {"align":"wide","className":"is-style-columns-1","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|60","left":"var:preset|spacing|50"}}}} -->
+<div class="wp-block-columns alignwide is-style-columns-1"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:quote {"className":"is-style-plain","style":{"typography":{"fontStyle":"normal","fontWeight":"400"},"spacing":{"blockGap":"var:preset|spacing|40"}},"fontSize":"large"} -->
+<blockquote class="wp-block-quote is-style-plain has-large-font-size" style="font-style:normal;font-weight:400"><!-- wp:group {"style":{"spacing":{"padding":{"top":"0","bottom":"0","left":"0","right":"0"},"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained","justifyContent":"left","contentSize":"400px"}} -->
+<div class="wp-block-group" style="margin-top:0;margin-bottom:0;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0"><!-- wp:paragraph {"style":{"typography":{"lineHeight":"1.1"}}} -->
```

Comment:
This is just to prevent overly long lines of text when the columns stack:

<img width="503" alt="image" src="https://github.com/user-attachments/assets/d20d8de5-33f0-4eff-91b5-c44523e3ecf0">

The width is set to match how the other quote patterns wrap their text in the design:

<img width="545" alt="image" src="https://github.com/user-attachments/assets/a48129be-35a5-4539-9326-a6148ffdfb9d">

Reply:
Thanks, I replied [here](https://github.com/WordPress/twentytwentyfive/pull/118/files#r1756563819) so that we can centralize the discussion.